Sources/IosAnalyticsDebugger/Util.m

```diff
@@ -27,23 +27,26 @@ +(BOOL) eventHaveErrors:(DebuggerEventItem *)event; {
 }
 
 + (NSInteger) barBottomOffset {
-    BOOL hasNotch = [self statusBarHeight] > 24.0;
-    
-    if (hasNotch) {
-        return 30;
+    if (@available(iOS 11.0, *)) {
+        return [UIApplication sharedApplication].keyWindow.safeAreaInsets.bottom;
```

Comment:
Should use safe area bottom inset if available. Keeping the old calculation for iOS 10.0 and lower.